parlai/utils/misc.py

```diff
@@ -384,7 +386,7 @@ def nice_report(report) -> str:
     try:
         import pandas as pd
 
-        use_pandas = True
+        use_pandas = False
```

Comment:
Oof - why false now? Don't we want to use pandas if we can import it?

Reply:
Debugging haha